app/views/posts/show.html.erb

```diff
@@ -1,3 +1,18 @@
+<% if user_signed_in? %>
+  <div>Signed in as... <%= current_user.email %></div>
+<%end%>
 <h1><%= @post.title %></h1>
-<h2><%= @post.author %></h2>
+<h3><%= @post.author %></h3>
+<h4>spice level: <%= @post.likes.count %></h4>
+<% if user_signed_in? %>
+  <% pre_like = @post.likes.find { |like| like.user_id == current_user.id} %>
+  <% if pre_like %>
+    <%= button_to 'dilute', post_like_path(@post, pre_like), method: :delete %>
```

Comment:
is this a separate button to unlike? in the future let's just click the same button again to unlike